recipes/rpclib/all/test_package/conanfile.py

```diff
@@ -0,0 +1,15 @@
+import os
+from conans import ConanFile, CMake, tools
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
```

Comment:
```suggestion
    generators = "cmake", "cmake_find_package_multi"
```

it will generate rpclib-config.cmake with `rpclib::rpc` target, just like the upstream